commons/all/src/main/java/org/infinispan/commons/util/Version.java

```diff
@@ -150,7 +157,7 @@ public static void main(String[] args) {
     * Prints full version information to the standard output.
     */
    public static void printFullVersionInformation() {
-      System.out.println(INSTANCE.brandname);
+      System.out.println(INSTANCE.brandName);
       System.out.println();
       System.out.printf("Version: \t%s%n", INSTANCE.version);
```

Comment:
use the `brandVersion` instead?
```suggestion
      System.out.printf("Version: \t%s%n", INSTANCE.brandVersion);
```